.github/workflows/publish-package.yml

```
@@ -0,0 +1,29 @@
+name: publish-package
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: node
+          package-name: "@ubiquitydao/rpc-handler"
+          default-branch: main
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "20.10.0"
+          registry-url: https://registry.npmjs.org/
+      - run: |
+          yarn install --immutable --immutable-cache --check-cache
+          yarn build
+          yarn pack
+          yarn publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_ACCESS_TOKEN }}
```

```suggestion
          NPM_ACCESS_TOKEN: ${{ secrets.NPM_ACCESS_TOKEN }}
```

This property name would make more sense. Why did you call it node auth token?

---

- https://github.com/ubiquibot/configuration/blob/development/.github/workflows/publish-package.yml

It's a copy paste from here

---

I think we should change them to a better property name.